_scripts/instruction-widget/templates/install/snap.html

```diff
@@ -18,6 +18,14 @@ <h3>Snap Support</h3>
     <a class="link-button" href='https://snapcraft.io/docs/installing-snapd/'>install snapd</a>
     </p>
 </li>
+<li>
+  Ensure that your version of snapd is up to date
+  <p>
+  Execute the following instruction on the command line on the machine to ensure
+  that you have the latest version of <tt>snapd</tt>.
+  </p>
+  <pre>sudo snap install core; sudo snap refresh</pre>
```

Comment:
```suggestion
  <pre>sudo snap install core; sudo snap refresh core</pre>
```
This is definitely a nitpick, but I think we should just refresh the core snap instead of taking the time to try and refresh every snap on their system.

I double checked that this syntax works on Debian 9 even before the `core` snap is installed and it does.

Reply:
Alright. In this case I also update https://github.com/certbot/certbot/pull/8313 to keep the instructions consistent between the website and certbot.

Reply:
Thanks! I personally think that makes sense.